crawler/config/defaults.py

```diff
@@ -28,53 +28,56 @@
 # sftp_root_write: directory on sftp in which to upload master files
 # file_names_to_ignore: array of files to exclude from processing, such as those
 #                       containing invalid headers
+CENTRE_REGEX_BARCODE = r"^[\W_]*([\w-]*)_([a-z]\d{2})[\W_]*$"
+CENTRE_DIR_BACKUPS = "data/backups"
+CENTRE_REGEX_SFTP_FILE = r"sanger_report_(\d{6}_\d{4}).*\.csv$"
```

Comment:
Looks like you can use this variable in all the centres, not just the first one

Reply:
Thanks, yeah I was testing if it would work and forgot to add it to the rest.